internal/locale/locales/en-us.yaml

```
@@ -1493,5 +1495,6 @@ uploadingredient_success:
     Name: [ACTIONABLE]{{.V0}}[/RESET]
     Namespace: [ACTIONABLE]{{.V1}}[/RESET]
     Version: [ACTIONABLE]{{.V2}}[/RESET]
+    Revision: [ACTIONABLE]{{.V3}}[/RESET]
 
-    You can install this package by running `[ACTIONABLE]state install {{.V1}}/{{.V0}} --ts now`[/RESET].
+    You can install this package by running `[ACTIONABLE]state install {{.V1}}/{{.V0}} --r {{.V3}} --ts now`[/RESET].
```

```suggestion
    You can install this package by running `[ACTIONABLE]state install {{.V1}}/{{.V0}} --r {{.V2}} --ts now`[/RESET].
```

It should be V2 right?

---

No, V2 holds the version, which is used above this line (it's a multiline locale).